### ISSUE TITLE
Cloudflare Site Value

### ISSUE THREAD

**Issue by stdavis** [gr2m/cloudflare-worker-github-oauth-login#2]
When I sign up for a new Cloudflare account, what do I put in as a site domain? This seems to be required to get my zone Id. I didn't think that I needed to host a domain on Cloudflare as it is only used to host a service.

![image](https://user-images.githubusercontent.com/1326248/67312338-6b241080-f4be-11e9-91dc-c7273178aa43.png)

**Comment by gr2m** [comment]
Hm I'm not sure that is necessary when only using workers? But good that you bring that up, I should add it to the documentation. Can you get around it? If not, can you enter `<your login>.workers.dev`?

**Comment by stdavis** [comment]
Without `CLOUDFLARE_ZONE_ID` defined in secrets, I get this error message from serverless in the deploy action:
![image](https://user-images.githubusercontent.com/1326248/67314121-67de5400-f4c1-11e9-8f4e-6518440c9961.png)

As for adding my `workers.dev` as a site, it didn't like it either:
![image](https://user-images.githubusercontent.com/1326248/67314252-9fe59700-f4c1-11e9-8dcd-5f1947f5a085.png)

**Comment by gr2m** [comment]
Hmmm I just entered `martynus.net`, one of my personal domains. Could you maybe ask support what they suggest to do?

**Comment by stdavis** [comment]
https://github.com/cloudflare/serverless-action/issues/3

**Comment by stdavis** [comment]
I wonder if this is the issue? https://github.com/cloudflare/serverless-cloudflare-workers/issues/36

This is what I got back from support:
>If you only have a Workers subdomain on Cloudflare then you will not have a proper Zone ID. A workaround if you are trying to use Wrangler would be to set workers_dev = true in the TOML file so the wrangler does not look for an account ID when you deploy. 

Does serverless use wrangler internally?

**Comment by gr2m** [comment]
I don't think so. I think serverless is a general-purpose tool for serverless apps, which includes a deploy commands, while wrangler is specific to CloudFlare.
